scripts/golos-boot-sequence/golos-boot-sequence.py

```diff
@@ -231,11 +231,12 @@ def createGolosAccounts():
     updateAuth('gls.vesting', 'active', 'owner', [args.public_key], ['gls.vesting@cyber.code'])
     updateAuth('gls.social',  'active', 'owner', [args.public_key], ['gls.publish@cyber.code'])
     updateAuth('gls.emit',    'active', 'owner', [args.public_key], ['gls.emit@cyber.code'])
+    updateAuth('gls.referral','active', 'owner', [args.public_key], ['gls.referral@cyber.code'])
```

Comment:
This authority gives the right to call contract actions from the smart contract code. In this case, this is not required (the `referral` smart-contract does not cause its other actions from the code). You need to create a new authority, as it is done for `cyber::createuser` (line 236) and link it with `gls.referral::addreferral` method.